src/js/toggler.js

```diff
@@ -28,9 +28,17 @@ $(() => {
     const checkedList = list.filter(chkBox => chkBox.checked)
 
     if (checkedList.length > 0) {
-      list.forEach(chkBox => { chkBox.checked = false })
+      list.forEach(chkBox => {
+        if (chkBox.checked) {
+          chkBox.click()
+        }
+      })
     } else {
```

Comment:
with your change we don't need to have `else` because the browser will handle that so:
```js
list.forEach(chkBox => {
  chkBox.click()
})
```
that's all